zerver/lib/email_notifications.py

```diff
@@ -154,13 +164,15 @@ def build_message_payload(message: Message) -> Dict[str, str]:
         html = message.rendered_content
         html = relative_to_full_url(user_profile.realm.uri, html)
         html = fix_emojis(html, user_profile.realm.uri, user_profile.emojiset)
-
+        if sender is not None:
+            plain, html = append_sender_to_message(plain, html, sender)
+        # do some shit here
```

Comment:
this comment is probably misplaced :)

Reply:
Really.Sorry. I wrote this down so that I know where to make changes later after reading the code. Was not really in a great mood. Forgot to clean it up.